CrackingTheCodingInterview/Code/Chapter 1/Question1_3.cs

```diff
@@ -50,5 +50,39 @@ public static void ReplaceSpaces(char[] inputString, int length)
                 }
             }
         }
+
+        // Space: O(N)
+        // Time: O(N)
+        public static string ReplaceSpaces(char[] inputString)
```

Comment:
This works if we're looking for another string, but in this case the question is asking for us to update the string in the original char array.